man/systemd.service.xml

```diff
@@ -978,8 +978,10 @@
         <listitem><para>Takes a list of exit status definitions that,
         when returned by the main service process, will force automatic
         service restarts, regardless of the restart setting configured
-        with <varname>Restart=</varname>. The argument format is
-        similar to
+        with <varname>Restart=</varname>. The process systemd 
```

Comment:
A bit more details on the expectations would be helpful I think, maybe phrase it something like:

> For <varname>Type=forking</varname> services the main
> process is the one forked by the service, not the one started
> by systemd. If the service never forks a process, there will be
> no main process as far as systemd is concerned.

Does that make it a bit clearer?

Reply:
@boucman would you mind having a look as well? Thanks!

Reply:
Honestly, instead of your phrasing, I would point out that what systemd considers the main process depends greatly on the Type= attribute and refer to that part of the man page instead of specifically talking about Type=forking